runtime-light/stdlib/array/array-functions.h

```diff
@@ -5,16 +5,201 @@
 #pragma once
 
 #include <concepts>
+#include <cstdint>
 #include <functional>
 #include <utility>
 
 #include "runtime-common/core/runtime-core.h"
+#include "runtime-common/stdlib/array/array-functions.h"
 #include "runtime-light/coroutine/task.h"
 #include "runtime-light/stdlib/math/random-functions.h"
 #include "runtime-light/utils/concepts.h"
 
+namespace {
```

Comment:
Anonymous namespaces in header files can lean to name clashing. Let's too its content into `array_functions_impl_` namespace

Reply:
fix